Doc/whatsnew/3.7.rst

```diff
@@ -1157,6 +2077,13 @@ Changes in Python behavior
   exceptions.
   (Contributed by Yury Selivanov in :issue:`32670`.)
 
+* :keyword:`async` and :keyword:`await` names are now reserved keywords.
```

Comment:
Please move this at the first place, as it is the most visible breaking change. Would be nice to add recommendations for solving problems caused by this change.